src/Components/CreateImageWizard/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -807,6 +810,15 @@ export const UsersList = () => {
           <TextListItem component={TextListItemVariants.dd}>
             {userPassword ? userPassword : 'None'}
           </TextListItem>
+          <TextListItem
+            component={TextListItemVariants.dt}
+            className="pf-v5-u-min-width"
+          >
+            SSH Key
```

Comment:
Let's use title case for the sake of consistency, otherwise looks good

```suggestion
            SSH key
```

Reply:
<img width="452" alt="Screenshot 2025-01-14 at 13 54 21" src="https://github.com/user-attachments/assets/5ada7653-047e-4632-8a03-18b3bfbe6a9b" />
in the task itself it looks like capital letter - https://issues.redhat.com/browse/HMS-4906
am I wrong?

Reply:
You're not wrong, it looks like capital letter. The mock should use title case as well for consistency sake. Like on the step itself

![image](https://github.com/user-attachments/assets/0527159c-04c5-4400-abed-70440e0a5b47)

Reply:
My bad. Keep in mind mocks aren't perfect. The engineer who implements the feature from the mocks should always look at the mocks with a critical eye - especially if they are quickly hacked together by an engineering manager :flushed: : 

For future reference, Patternfly has guidelines around capitalization:
https://www.patternfly.org/ux-writing/capitalization